packages/components/src/autocomplete/index.tsx

```diff
@@ -444,8 +444,14 @@ export function useAutocompleteProps( options: UseAutocompleteProps ) {
 			function _onKeyDown( event: KeyboardEvent ) {
 				onKeyDownRef.current?.( event );
 			}
+			// To do: comment.
```

Comment:
Do it 😄

Reply:
Looks like this is no longer needed after other fixes.